pkg/volume/azure_dd/azure_dd.go

```diff
@@ -164,7 +165,9 @@ func (plugin *azureDataDiskPlugin) GetVolumeLimits() (map[string]int64, error) {
 	}
 
 	if vmSizeList == nil {
-		result, err := az.VirtualMachineSizesClient.List(context.TODO(), az.Location)
+		ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
+		defer cancel()
```

Comment:
Should we also add same timeout to getAllStorageAccounts?

Reply:
@feiskyer fixed.